pages/sections.tsx

```diff
@@ -37,6 +39,78 @@ export const SectionsPage: NextPage = () => {
     "tenant"
   ]);
 
+  const idAndResidencyStartedValidator = (
+    values: [
+      StoreValue<
+        ProcessDatabaseSchema["schema"],
+        StoreNames<ProcessDatabaseSchema["schema"]>
```

Comment:
```suggestion
        "id" | "residency" | "tenant"
```

I think this should work, and it's a bit clearer about what it's doing.